rdmo/projects/viewsets.py

```diff
@@ -334,6 +334,11 @@ def visibility(self, request, pk=None):
             throttle_classes=[EmailThrottle])
     def contact(self, request, pk):
         if settings.PROJECT_CONTACT:
+            try:
+                project = self.get_object()
```

Comment:
The `self.get_object()` (permission handling) is needed by the `try ... except` is not. Since I am not sold on the caching thing yet, I will fix this in the original branch.